Source/DiabloUI/selgame.cpp

```diff
@@ -448,7 +448,7 @@ void selgame_Password_Select(int value)
 	m_game_data->bTheoQuest = sgOptions.Gameplay.bTheoQuest;
 	m_game_data->bCowQuest = sgOptions.Gameplay.bCowQuest;
 
-	if (SNetCreateGame(nullptr, selgame_Password, nullptr, 0, (char *)m_game_data, sizeof(GameData), MAX_PLRS, nullptr, nullptr, gdwPlayerId)) {
+	if (SNetCreateGame(nullptr, selgame_Password, (char *)m_game_data, sizeof(m_game_data), gdwPlayerId)) {

```

Comment:
reinterpret_cast instead of c-style cast

Reply:
How about we save that for when we run the rules that deal with casting :)